guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Is the dataset containing 5 providers with 10 branch locations? If so, would be good to also show the column that shows provider names.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I used the wrong csv so it shows 10 entries. I will update the code cell to reflect correctly, I have also added a fictitious name column so we have the names as well.</p>